src/main/java/com/example/library/validation/FieldMatchValidator.java

```diff
@@ -0,0 +1,16 @@
+package com.example.library.validation;
+
+import com.example.library.dto.user.UserRegistrationRequestDto;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+
+public class FieldMatchValidator implements ConstraintValidator<FieldMatch,
+        UserRegistrationRequestDto> {
+    @Override

```

Comment:
Do you have checkstyle?
```suggestion
        UserRegistrationRequestDto> {
        
    @Override
```